src/main.js

```diff
@@ -35,112 +35,118 @@ startButton.addEventListener('click', startCountDown);
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeMeditateColor() {
-    changeColor(meditateButton, 'meditate-active', studyButton, 'study-active', exerciseButton, 'exercise-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(meditateButton, 'meditate-active');
+  removeClassProperty(studyButton, 'study-active', exerciseButton, 'exercise-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeExerciseColor() {
-    changeColor(exerciseButton, 'exercise-active', studyButton, 'study-active', meditateButton, 'meditate-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(exerciseButton, 'exercise-active');
+  removeClassProperty(studyButton, 'study-active', meditateButton, 'meditate-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function limitTimeInput(timeInput, num) {
   preventInvalids(event, timeInput);
   timeInput.value = parseInt(timeInput.value);
   if (timeInput.value > num) {
-      timeInput.value = num;
+    timeInput.value = num;
   }
 }
 
 function preventInvalids(event, inputField) {
-    var invalidChars = ['+', '-', 'e', 'E'];
-    for (var i = 0; i < invalidChars.length; i++) {
-        if (event.key === invalidChars[i]) {
-            event.preventDefault();
-            inputField.value = '';
-        }
+  var invalidChars = ['+', '-', 'e', 'E'];
+  for (var i = 0; i < invalidChars.length; i++) {
+    if (event.key === invalidChars[i]) {
+      event.preventDefault();
+      inputField.value = '';
     }
+  }
+}
+function updateErrorMsg(userInput, errorMsgIndex) {
+  userInput.value ? 
+  addClassProperty(errorMsg[errorMsgIndex], 'hidden') : removeClassProperty(errorMsg[errorMsgIndex], 'hidden');
 }
-
 function limitAccomplish() {
-  areInputsDefined([accomplishInput], 1);
+  updateErrorMsg(accomplishInput, 1);
 }
 
-function limitMin() { 
+function limitMin() {
   limitTimeInput(minutesInput, 90);
-  areInputsDefined([minutesInput], 2);
+  updateErrorMsg(minutesInput, 2);
 }
 
 function limitSec() {
   limitTimeInput(secondsInput, 59);
-  areInputsDefined([secondsInput], 3);
+  updateErrorMsg(secondsInput, 3);
 }
 
-function createCurrentActivity() {
-    changeColor(newActivitySection,'hidden', currentActivitySection, 'hidden');
-    if (studyButton.classList.contains('study-active')) {
-        createInstance('study-active');
-        changeColor(startButton,'study-circle');
-    } else if (meditateButton.classList.contains('meditate-active')) {
-        createInstance('meditate-active');
-        changeColor(startButton,'meditate-circle');
-    } else if (exerciseButton.classList.contains('exercise-active')) {
-        createInstance('exercise-active');
-        changeColor(startButton,'exercise-circle');
-    }
+function updateCircleColor(button, classProperty) { 
```

Comment:
nice, this function is pretty reusable with that interpolation in the parameter.